.cirrus.yml

```diff
@@ -180,7 +180,8 @@ sanity_task:
   sanity_script:
     - source cirrus-env QA
     - source set_maven_build_version $BUILD_NUMBER
-    - mvn clean compile --projects java-checks-test-sources --also-make-dependents
+    - mvn clean install -Dmaven.test.skip=true
+    - mvn compile --projects java-checks-test-sources --also-make-dependents
```

Comment:
The command that was used before, `mvn clean compile --projects java-checks-test-sources --also-make-dependents`, if executed before compiling the whole project, fails because of compilation errors (GeneratedCheckList class doesn't exist).
Now, I updated the script a bit to fix it without building the whole project (see the latest version of this file).
Also, yes, the documentation needed a minor update, and it has been added.